Source/KerbalCombatSystems/EscapePod.cs

```diff
@@ -89,35 +120,14 @@ private IEnumerator RunEscapeSequence()
             yield break;
         }
 
-        public override void OnStart(StartState state)
-        {
-            if (!HighLogic.LoadedSceneIsFlight) return;
-
-            //create the appropriate lists
-            AIPartList = new List<Part>();
-            List<ModuleShipController> AIModulesList;
-
-            //designate ship that's being escaped from
-            Parent = vessel;
-
-            //find ai parts and add to list
-            AIModulesList = vessel.FindPartModulesImplementing<ModuleShipController>();
-            foreach (var Controller in AIModulesList)
-            {
-                AIPartList.Add(Controller.part);
-            }
-
-            StartCoroutine(StatusRoutine());
-        }
-
         IEnumerator StatusRoutine()
         {
-            if (!Escaped)
+            if (!escaped)
             {
                 CheckConnection();
             }
 
-            yield return new WaitForSeconds(5f);
+            yield return new WaitForSeconds(refreshRate);
             StartCoroutine(StatusRoutine());
```

Comment:
You could make this more readable and possibly safer by using a while loop instead.